sdk/iot/Azure.Iot.Hub.Service/src/Generated/CloudToDeviceMessagesRestClient.cs

```diff
@@ -157,8 +157,8 @@ internal HttpMessage CreateAbandonFeedbackNotificationRequest(string lockToken)
             return message;
         }
 
-        /// <summary> This method abandons a feedback message. The lockToken obtained when the message was received must be provided to resolve race conditions when abandoning, a feedback message. A abandoned message is deleted from the feedback queue. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-messaging for more information. For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version &apos;2020-03-13&apos;.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version &apos;2019-10-01&apos;. </summary>
-        /// <param name="lockToken"> Lock Token. </param>
+        /// <summary> Abandons a cloud-to-device feedback message. An abandoned message is deleted from the feedback queue of the service. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-messaging for more information. </summary>
```

Comment:
This description needs to be updated: 
`An abandoned message is enqueued back to the service's feedback queue.`